assets/tests/hooks/usePersistedStateReducer.test.ts

```diff
@@ -89,25 +88,6 @@ describe("usePersistedStateReducer", () => {
     expect(state).toEqual(expectedState)
   })
 
-  test("stores persisted keys in localstorage when they change", () => {
```

Comment:
Is localstorage still used for anything else besides the selected vehicle?

Reply:
Oh...yes, I think so. I'll restore the balance of the test.